scripts/plots/rtofs/exevs_rtofs_argo_grid2obs_plots.sh

```diff
@@ -161,16 +161,6 @@ for levl in 0 50 125 200 400 700 1000 1400; do
   done
 done
 
-# Cat the plotting log files
-log_dir=$DATA/logs/rtofs
-log_file_count=$(find $log_dir -type f |wc -l)
-if [[ $log_file_count -ne 0 ]]; then
-    for log_file in $log_dir/*; do
-        echo "Start: $log_file"
-        cat $log_file
-        echo "End: $log_file"
-    done
-fi
 
```

Comment:
@SamiraArdani-NOAA You have removed the part of the plot scripts that cat's the plotting log files.  Please be advised that these should stay in these scripts.  The removal of this section was due to the use of LOG_TO_TERMINAL_ONLY METplus option, for those codes that use METplus.  Plotting does not use METplus, so there is no terminal log of your plot output.  Thus these still need to be included here.  

Please add these blocks back into your plot scripts.  Thanks!

Reply:
@PerryShafran-NOAA : I added those blocks back into the plot scripts.